addons/web/static/tests/views/calendar/calendar_view_tests.js

```diff
@@ -54,6 +55,16 @@ const uid = -1;
 
 QUnit.module("Views", ({ beforeEach }) => {
     beforeEach(() => {
+        const optionsProperty = Object.getOwnPropertyDescriptor(
+            CalendarYearRenderer.prototype,
+            "options"
+        );
+        patchWithCleanup(CalendarYearRenderer.prototype, {
+            get options() {
+                return { ...optionsProperty.get.call(this), weekNumbers: true };
+            },
+        });
```

Comment:
Do this only in the related tests.
```suggestion
        patchWithCleanup(CalendarYearRenderer.prototype, {
            get options() {
                return { ...this._super(), weekNumbers: true };
            },
        });
```

It will break in a forward port (will be replaced by `super.options`) but it's cleaner